man/pdp.rank.Rd

```diff
@@ -0,0 +1,44 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/plotter.R
+\name{pdp.rank}
+\alias{pdp.rank}
+\title{Given an interpreter object with choice of PDP ranking methodology
+       (default: 'Variance'), produce PDP 'ranking' scores by feature. Optionally,
+       permits a new observation to weight the PDP function and rankings.}
+\usage{
+pdp.rank(
+  object,
+  rank.method = "Variance",
+  new.obs = NULL,
+  feature = NULL,
+  qt = 5
+)
+}
+\arguments{
+\item{object}{The Interpreter class that we want understand the PDP ranking scores of.}
+
+\item{rank.method}{A string to select which PDP ranking methodology. Should be one of
+c("Variance", "FO.Derivative"). When set to "Variance" the PDP functions are ranked by variance
+of the function over the range of grid.points and when set to "FO.Derivative" the PDP functions
+are ranked by empirical first order derivative.}
```

Comment:
We should describe the "FO.Derivative" method better here. They are ranked by the maximum absolute value of the first order derivatives.